app/models/bestuursorgaan.js

```diff
@@ -1,20 +1,18 @@
-import Model from 'ember-data/model';
-import attr from 'ember-data/attr';
-import { belongsTo, hasMany } from 'ember-data/relationships';
+import Model, { belongsTo, hasMany, attr } from '@ember-data/model';
 
-export default Model.extend({
-  uri: attr(),
-  naam: attr(),
-  bindingStart: attr('date'),
-  bindingEinde: attr('date'),
-  bestuurseenheid: belongsTo('bestuurseenheid', { inverse: 'bestuursorganen' }),
-  classificatie: belongsTo('bestuursorgaan-classificatie-code', { inverse: null }),
-  isTijdsspecialisatieVan: belongsTo('bestuursorgaan', { inverse: 'heeftTijdsspecialisaties' }),
-  wordtSamengesteldDoor: belongsTo('rechtstreekse-verkiezing', { inverse: 'steltSamen' }),
-  heeftTijdsspecialisaties: hasMany('bestuursorgaan', { inverse: 'isTijdsspecialisatieVan' }),
-  bevat: hasMany('mandaat', { inverse: null }),
+export default class Bestuurorgaan extends Model {
+  @attr uri;
+  @attr naam;
+  @attr('date') bindingStart;
+  @attr('date') bindingEinde;
+  @belongsTo('bestuurseenheid', { inverse: 'bestuursorganen' }) bestuurseenheid;
+  @belongsTo('bestuursorgaan-classificatie-code', { inverse: null }) classificatie;
+  @belongsTo('bestuursorgaan', { inverse: 'heeftTijdsspecialisaties' }) isTijdsspecialisatieVan;
+  @belongsTo('rechtstreekse-verkiezing', { inverse: 'steltSamen' }) wordtSamengesteldDoor;
+  @hasMany('bestuursorgaan', { inverse: 'isTijdsspecialisatieVan' }) heeftTijdsspecialisaties;
+  @hasMany('mandaat', { inverse: null }) bevat;
 
-  rdfaBindings: { // eslint-disable-line ember/avoid-leaking-state-in-ember-objects
+  rdfaBindings = { // eslint-disable-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
Wrap this object in Object.freeze to resolve the linting error. Or since it's an ES6 class now, you can also convert it to a `get` function (same comment applies on other models)